packages/shared/src/components/squads/SquadsToShare.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement, MouseEvent, useMemo, useContext } from 'react';
+import { verifyPermission } from '../../graphql/squads';
+import { SourcePermissions, Squad } from '../../graphql/sources';
+import SourceProfilePicture from '../profile/SourceProfilePicture';
+import { SocialShareIcon } from '../widgets/SocialShareIcon';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { Origin } from '../../lib/analytics';
+import PlusIcon from '../icons/Plus';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import { useCreateSquadModal } from '../../hooks/useCreateSquadModal';
+import { ButtonSize } from '../buttons/Button';
+import { ProfileImageSize } from '../ProfilePicture';
+
+interface SquadsToShareProps {
+  isLoading?: boolean;
+  onClick: (e: MouseEvent, squad: Squad) => void;
+  size?: ButtonSize;
+  squadIconSize?: ProfileImageSize;
+}
+
+export function SquadsToShare({
+  isLoading,
+  onClick,
+  size = ButtonSize.Large,
+  squadIconSize = 'xlarge',
```

Comment:
This could probably be the IconSize enum right?

Reply:
Unfortunately, they are not an `enum` yet. It would be a big change to start here. We should probably write a ticket to migrate. WDYT?

Reply:
Ah sorry it sounded like a `IconSize` but it's a `ProfileImageSize` perhaps we should rename this then?
It's not an icon but a squadImage rather? (Just to avoid confusion down the line)

Reply:
Ah, that is true. Will update quickly.